src/ex4/server/helper/logger.js

```diff
@@ -0,0 +1,11 @@
+import { writeToFile, readFile } from './fileHandler.js'
+
+export const logger = async (req, res, next) => {
+    const currentDate = new Date().toJSON().slice(0, 10).replace(/-/g, '/');
```

Comment:
Too many operation to get the format you want. (while `.toJSON` is a heavy operation by itself)
You can just do:
`new Date().toLocaleDateString()`